WcaOnRails/lib/tasks/work.rake

```diff
@@ -7,6 +7,6 @@ namespace :work do
     SubmitReportNagJob.perform_later
     ComputeLinkings.perform_later
     DumpDeveloperDatabase.perform_later
-    SyncMailingListsJob.perform_later
+    SyncMailingListsJob.perform_later if ENVied.WCA_LIVE_SITE
```

Comment:
nit: It might be worth adding a comment explaining why we don't want to do this on the "live site"?